README.md

```diff
@@ -235,7 +237,6 @@ end
 
 When creating a metric, ActiveReporting will recognize implicit hierarchies for this dimension. The hierarchies correspond to the [values](https://www.postgresql.org/docs/8.1/static/functions-datetime.html#FUNCTIONS-DATETIME-TRUNC) supported by PostgreSQL. (See example under the metric section, below.)
 
-*NOTE*: PRs welcomed to support this functionality in other databases.
```

Comment:
I guess PRs are still welcome to support other databases 😉

Reply:
LOL, true but the rest of databases are SQLitle which nobody use in production. I doubt someone will do that